web_fragments/views.py

```diff
@@ -45,11 +44,11 @@ def render_to_standalone_html(self, request, fragment, **kwargs):  # pylint: dis
         Render the specified fragment to HTML for a standalone page.
         """
         template = get_template(STANDALONE_TEMPLATE_NAME)
-        context = Context({
```

Comment:
I guess we aren't using this view in edx-platform, since that code certainly wasn't working with Django 1.11.

Reply:
I'll assume this is a remark that does not require action. If you would like me to remove this view because it is dead code, please let me know.